admin-dev/themes/new-theme/js/pages/product/edit/product-type-manager.js

```diff
@@ -51,22 +52,25 @@ export default class ProductTypeManager {
    */
   confirmTypeSubmit() {
     let confirmMessage = this.$typeSelector.data('confirm-message');
-    let confirmWarning;
+    let confirmWarning = '';
 
-    switch (this.initialType) {
-      case ProductMap.productType.COMBINATIONS:
-        confirmWarning = this.$typeSelector.data('combinations-warning');
-        break;
-      case ProductMap.productType.PACK:
-        confirmWarning = this.$typeSelector.data('pack-warning');
-        break;
-      case ProductMap.productType.VIRTUAL:
-        confirmWarning = this.$typeSelector.data('virtual-warning');
-        break;
-      case ProductMap.productType.STANDARD:
-      default:
-        confirmWarning = '';
-        break;
+    // If no productId we are in creation page so no need for extra warning
+    if (this.productId) {
+      switch (this.initialType) {
+        case ProductMap.productType.COMBINATIONS:
+          confirmWarning = this.$typeSelector.data('combinations-warning');
+          break;
+        case ProductMap.productType.PACK:
+          confirmWarning = this.$typeSelector.data('pack-warning');
+          break;
+        case ProductMap.productType.VIRTUAL:
+          confirmWarning = this.$typeSelector.data('virtual-warning');
+          break;
+        case ProductMap.productType.STANDARD:
+        default:
+          confirmWarning = '';
+          break;
```

Comment:
```suggestion
```

Reply:
ESLint doesn't accept it, already tried 🤷‍♂️